src/main/java/com/amazonaws/kinesisvideo/producer/ClientInfo.java

```diff
@@ -11,16 +11,31 @@ public class ClientInfo {
     /**
      * Current version for the structure as defined in the native code
      */
-    public static final int CLIENT_INFO_CURRENT_VERSION = 0;
+    public static final int CLIENT_INFO_CURRENT_VERSION = 2;
     public static final int DEFAULT_LOG_LEVEL = 4;
 
+    public static enum automaticStreamingFlags {
+        AUTOMATIC_STREAMING_INTERMITTENT_PRODUCER(0), AUTOMATIC_STREAMING_ALWAYS_CONTINUOUS(256);
+        private final int value;
```

Comment:
Nit: Can be specific. `streamingFlagValue` or something similar.

Reply:
Changed to the suggested name in the latest commit